### PR TITLE
Rebase render tests' expected images

### PR DESCRIPTION
**Pull request by stefankarschti**
Rebase render tests to Drawable GL actuals:
- heatmap failures from #1543 
- combinations translucent--opaque #1511 
- a pixel diff regression 2534

### CODE REVIEW

**Review by TimSylvester — APPROVED** [review]

**Review by louwers — APPROVED** [review]

**Review by alexcristici — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -44,4 +44,8 @@ node_modules
 /bazel-*
 CMakeLists.txt.user
 CMakeFiles
-__generated__
\ No newline at end of file
+__generated__
+
+/metrics/integration/query-tests/**/actual.json
+/metrics/integration/render-tests/**/actual.png
+/metrics/integration/render-tests/**/diff.png
```
Comment (alanchenboy) [inline .gitignore:51]: Can we ignore all actual.png and diff.png, cause there are same files under the test directory.

Comment (stefankarschti) [reply .gitignore:51]: I guess we could...

Comment (stefankarschti) [reply .gitignore:51]: I don't see them in my changed files usually, what is the test program that generates them?

Comment (stefankarschti) [reply .gitignore:51]: Let's add the other ignores in a different PR...

Comment (alanchenboy) [reply .gitignore:51]: For example `/Users/wei.chen/work/maplibre/test/fixtures/annotations/add_multiple ` has actrual.png and diff.png, after cpp unit test(mbgl-test).

Comment (stefankarschti) [reply .gitignore:51]: There is already a [test/fixture/.gitignore](https://github.com/maplibre/maplibre-native/blob/main/test/fixtures/.gitignore) that has them 
```
diff.png
actual.png
*.db
```

**Pull request merged**